<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabelTrainer</class>
 <widget class="QDialog" name="TabelTrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/logo.ico</normaloff>:/img/logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Opgaver:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSolved">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labAfMax">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> af </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labAntal">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbColors">
       <property name="text">
        <string>Vis rettelser l√∏bende</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labFirst">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSign">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSecond">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labEquals">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leResult">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Afslut med ENTER</string>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>550</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row>
      <property name="text">
       <string notr="true">1</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true">10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">7</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">8</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">9</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">10</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labElapsed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOnceMore">
       <property name="text">
        <string>En gang til</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Afslut</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
